jscomp/core/lam_dispatch_primitive.ml

```diff
@@ -705,7 +705,7 @@ let translate loc (prim_name : string)
       begin match args with 
         | [ tag; 
             {expression_desc = Number (Int { i ;_}); _} ] ->
-          E.make_block tag Blk_na 
+          E.make_block tag (Blk_na "")
```

Comment:
I would always pass *some* debug string to make it easier to track in future.

Reply:
Indeed, will add a `obj_block` for debugging